### ISSUE TITLE
what caused a sharp decrease(except for atlasnet_singleview_25_squares which is increased) of loss near about 120 epoch?

### ISSUE THREAD

**Issue by sumai-1998** [ThibaultGROUEIX/AtlasNet#67]
just as shown in every pre-traind models folder, this sharp loss change seems unusual
atlasnet_singleview_25_squares
atlasnet_singleview_1_squares
atlasnet_autoencoder_25_squares
atlasnet_autoencoder_1squares

curve_log

I also trained the model myself, also has the same loss change

**Comment by sumai-1998** [comment]
> just as shown in every pre-traind models folder, this sharp loss change seems unusual
> atlasnet_singleview_25_squares
> atlasnet_singleview_1_squares
> atlasnet_autoencoder_25_squares
> atlasnet_autoencoder_1squares
> 
> curve_log
> 
> I also trained the model myself, also has the same loss change


just wonder why, has you ever consider about this before?

**Comment by ThibaultGROUEIX** [comment]
Yes, it is a learning rate decay.
Best regards,
Thibault

**Comment by sumai-1998** [comment]
> Yes, it is a learning rate decay.
> Best regards,
> Thibault
